src/command/run_test.go

```diff
@@ -14,7 +14,7 @@ import (
 
 func TestCommand_Run(t *testing.T) {
 	res, err := command.Run("echo", "foo")
-	assert.Nil(t, err)
+	assert.NoError(t, err)
```

Comment:
Nice idea to use `NoError` here! I wasn't aware of this. I'm going to change this everywhere in the codebase.

Reply:
I already did a global replace in the second commit.